src/openzaak/components/catalogi/admin/admin_views.py

```diff
@@ -22,7 +24,22 @@
 )
 
 
-class CatalogusZaakTypeImportUploadView(PermissionRequiredMixin, FormView):
+class AdminContextMixin:
```

Comment:
Please put this in `openzaak.utils.admin` so that we don't end up with 4 copies of the same thing in different components/apps again :grimacing: